src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,82 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+
+type Props = {
```

Comment:
```suggestion
interface PaginationProps {
```

Better to use `interface` for object